test/metabase/util/malli/registry_test_macro.cljc

```diff
@@ -0,0 +1,18 @@
+(ns metabase.util.malli.registry-test-macro
```

Comment:
I think for a pure-macros namespace like this it might actually be easier to create a separate `.clj` and `.cljs` version of the namespace -- then the `.clj` version doesn't need `#?(:clj ...)` around everything and the `.cljs` is just a shim with `:require-macros`. Maybe not important here since it's so small but just some food for thought

Reply:
Good idea, I'll do it this way